build_tools/setup_helpers/extension.py

```diff
@@ -57,6 +60,10 @@ def run(self):
         super().run()
 
     def build_extension(self, ext):
+        if ext.name == 'torchaudio.libtorchaudio':
+            # libtorchaudio is built as a part of _torchaudio
```

Comment:
per offline discussion, improve comment to indicate that the main purpose of this if statement is to avoid building extension multiple times (as opposed to specifically not building it when called with `libtorchaudio`)